### PR TITLE
BGP module update.

### PR DESCRIPTION
**Pull request by plorinquer**
This PR, although not as complete as I would like it to be, improves the existing BGP module.
Unit tests are provided in bgp.uts

### CODE REVIEW

**Comment by gpotter2** [comment]
@plorinquer please fix the appveyor (windows) tests

File: scapy/contrib/bgp.py [hunk 1]
```diff
@@ -1,170 +1,3909 @@
...
+# Multiprotocol Extensions for BGP-4
+#
+
+class BGPCapMultiprotocol(BGPCapability):
+    """
+    This class provides an implementation of the Multiprotocol
+    capability.
+    """
+
+    #________________________________________________________________________
```
Comment (gpotter2) [inline scapy/contrib/bgp.py]: We don't need the RFCs inside the code... simply link them

**Comment by p-l-** [comment]
Looks like you still have a `socket.inet_ntop()` call that breaks Windows tests.

**Comment by gpotter2** [comment]
@plorinquer On Windows, i got this:
```
>>> a.nlri[18].prefix
'::ffff::/96'
```
so this:
`a.nlri[18].prefix == "::ffff:0.0.0.0/96"`
fails

**Comment by guedou** [comment]
@gpotter2 that's a bug on Windows. The current inet_ntop implementation is not perfect =\

**Comment by guedou** [comment]
@p-l- this PR is good for me. Do you want to merge it ?

**Comment by plorinquer** [comment]
Thanks !

**Comment by p-l-** [comment]
@plorinquer thank **you**!

**Pull request merged**